htsget-http-actix/Cargo.toml

```diff
@@ -1,15 +1,15 @@
 [package]
 name = "htsget-http-actix"
 version = "0.1.0"
-authors = ["Daniel del Castillo de la Rosa <delcastillodelarosadaniel@gmail.com>"]
+authors = ["Daniel del Castillo de la Rosa <delcastillodelarosadaniel@gmail.com>", "Marko Malenic <mmalenic1@gmail.com>"]
 edition = "2018"
 
 [features]
 default = ["async", "htsget-search/default", "htsget-http-core/default"]
 async = ["tokio", "futures", "htsget-search/async", "htsget-http-core/async"]
 
 [dependencies]
-actix-web = "3"
+actix-web = { version = "4.0.0-beta.13" }
```

Comment:
How close do you feel actix are to finalising 4.0.0?

Reply:
Unfortunately I have no idea. They seem to have quite a few versions, and may be close to finalising it?

Reply:
Thanks a ton for those fixes Marco! Andrew: beta #ftw if it works, right? Are you worried about stability?... perhaps in a subcrate/future I'd go with sth like [Axum](https://docs.rs/axum/0.4.2/axum/index.html), which seems to fit our puzzle even better than Actix.

Reply:
If its due next week - I'd wait.. but yes if they don't really have a timetable then beta is fine..